src/cluster_legacy.h

```diff
@@ -289,6 +314,24 @@ static_assert(offsetof(clusterMsg, data) == 2256, "unexpected field offset");
                                               primary is up. */
 #define CLUSTERMSG_FLAG0_EXT_DATA (1 << 2) /* Message contains extension data */
 
+typedef struct {
+    char sig[4];     /* Signature "RCmb" (Cluster message bus). */
+    uint32_t totlen; /* Total length of this message */
+    uint16_t ver;    /* Protocol version, currently set to CLUSTER_PROTO_VER. */
+    uint16_t notused1;
+    uint16_t type; /* Message type */
+    union clusterMsgDataLight data;
+} clusterMsgLight;
```

Comment:
I am fine with `Light` but just wanted to throw another option for consideration. Would "compact" better capture the intent of this new message struct?
```suggestion
} clusterMsgCompact;
```

Reply:
Hi, IMO we are not compacting the struct but making it light/less fields. I would prefer Light.

Reply:
I prefer light too. Light-weight means it contains less data, not the same data in a more compact format.